### ISSUE TITLE
There is no notification when starts ringing 

### ISSUE THREAD

**Issue by myNameIsHamza** [trikita/talalarmo#33]
There is no notification or the app not lunched when it starts ringing

**Comment by Fluffkin** [comment]
As far as I can tell this is a new limitation in Android 10. It won't let the app display the "turn off alarm" screen over the lock screen. This means the alarm rings, but the only way I can stop it is to reboot the phone. Not ideal.

As there's been no code edits on this for 4 years, it looks like if you have Android 10, you need to find a different alarm app that does what Talalarmo did, but that actually works on Android 10.

**Comment by myNameIsHamza** [comment]
Yeah I see, but now working with flutter, I managed to make it work by adding these two activities in the AndroidManifest.xml:
`<activity
            android:showWhenLocked="true"
            android:turnScreenOn="true"
            >`
            
and I guess it will work in this app too.
